Week7/Solution/solution6.py

```diff
@@ -0,0 +1,11 @@
+from pandas import *
+
+Column=['Product No', 'Product Name', 'January', 'February', 'March', 'April', 'May', 'June']
+dtf = read_csv('Student_Data.csv')
+
+dtf.loc[2.5] = [116, 'Portable Hardisk',90,78,76,45,34,22]

```

Comment:
dtf.loc[2.5] means which row?

Reply:
we need to enter a new row between 2nd row and 3rd row so we need to write 2.5 to insert a new row between 2nd and 3rd row